packages/sources/src/styling/global/headings.scss

```diff
@@ -27,21 +43,33 @@ h5,
 .h5,
 h6,
 .h6 {
+  margin-bottom: $dso-vertical-spacing-medium;
   margin-top: ($line-height-computed * 0.5);
 }
 
+h4,
+.h4 {
+  line-height: $dso-font-line-height-h4;
+}
+
 h4 {
   color: $bosgroen;
   font-size: $dso-font-size-h4;
   font-weight: 600;
 }
 
+h5,
+.h5 {
+  line-height: $dso-font-line-height-h5;
+}
+
 h5 {
   color: $grijs-90;
   font-size: $dso-font-size-h5;
   font-weight: 600;
 }
 
-#{$headings} {
```

Comment:
Waar is deze styling naartoe verhuisd?

Reply:
Deze werden van de `@dso-toolkit` weer overschreven, en aangezien die waarden nu in dit bestand staan (regel 6 en 45), bleek dit stukje overbodig.

Reply:
Zie bijvoorbeeld de stijling op de h2 in https://dso-toolkit.nl/34.1.0/docs/fundament/typography.html:

![image](https://user-images.githubusercontent.com/5871330/160335060-7fd27eea-decf-49a9-817d-ddefd1eaffb7.png)